examples/demo/main.go

```diff
@@ -59,22 +66,40 @@ func main() {
 	})
 
 	bs := bytesub.New()
-	go func() {
-		i := 0
-		for range time.Tick(time.Second) {
-			bs.AnnounceLatest([]byte(fmt.Sprintf("Uptime: %d", i)))
-			i++
+	u := updater.New(context.Background(), func(b []byte) {
+		var out bytes.Buffer
+		err := json.Indent(&out, b, "", "  ")
+		if err == nil {
+			bs.AnnounceLatest(out.Bytes())
+		} else {
+			bs.AnnounceLatest(b)
 		}
-	}()
+	})
 
 	http.Handle("/connect", websocket.Handler(func(ws *websocket.Conn) {
 		bs.Subscribe(ws.Request().Context(), ws)
 	}))
 
 	logger.Info("Starting Server")
 
+	go startComponents(cfg, u)
+
 	// TODO: Other services read their port from the common config map, how should
 	// this be choosing the ports it exposes?
 	err = http.ListenAndServe(":51507", nil)
```

Comment:
read the port from kubectl client

Reply:
kubectl isn't included in our images?  How are you expecting me to do this?  (Also, this should maybe be a different pr?)